demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -171,6 +172,42 @@ def is_image_path_valid(self) -> bool:
             return False
         return True
 
+    def verify_no_empty_sections(self) -> bool:
+        """ Check the following:
+            1. if Troubleshooting exists, it isn't empty.
+            2. no 'FILL IN REQUIRED PERMISSIONS HERE'.
+            3. no unexplicit version number - such as "version xx of".
```

Comment:
please consider splitting to 3 smaller functions, each check should be in a separate func, they have nothing in common IMO.

Reply:
they all check that there are no "leftovers" from the default "read-me" demisto-sdk generates, so I think they do belong together.

Reply:
I see your point, so let's rename this func, and consider re writing it maybe using closures or in a different manner, as each check is an individual and a valid error.